tex/latex/biblatex/biblatex.def

```diff
@@ -2803,6 +2804,29 @@
 
 \newbibmacro*{date}{\printdate}
 
+\DeclareFieldFormat{patentextradates}{\mkbibparens{#1}}
+\newbibmacro*{patentdate}{
+  \printdate%
+  {%
+  \ifboolexpr{%
+      test {\iffieldundef{filedyear}} and%
+      test {\iffieldundef{grantedyear}} and%
+      test {\iffieldundef{expiredyear}}%
+    }%
+    {}%
+    {%
+      \newunit%
+      \printtext[patentextradates]{%
+          \iffieldundef{filedyear}{}{\bibstring{filed}\space \printfileddate}%
```

Comment:
This could be handled more elegantly with field formats, cf. `urldate` and `\DeclareFieldFormat{urldate}{\mkbibparens{\bibstring{urlseen}\space#1}}`.

Reply:
I do not understand this one